versioned/storage/bigtable/src/main/java/org/projectnessie/versioned/storage/bigtable/BigTableBaseBackendConfig.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.storage.bigtable;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+public interface BigTableBaseBackendConfig {
+
+  default String instanceId() {
+    return "nessie";
+  }
+
+  Optional<String> appProfileId();
+
+  Optional<String> quotaProjectId();
+
+  Optional<String> endpoint();
+
+  Optional<String> mtlsEndpoint();
+
+  Optional<String> emulatorHost();
+
+  Optional<String> tablePrefix();
```

Comment:
Aside from `tablePrefix()` these settings look like BT client settings. `BigTableBackend` does not appear to use them.

WDYT about keeping `tablePrefix()` separate and renaming this to `BigTableClientConfig`?

Reply:
> renaming this to BigTableClientConfig?

"backend" refers to the Nessie storage backend implementation - not some "BT backend"